docs/dictionary/function/number.lcdoc

```diff
@@ -14,6 +14,8 @@ Syntax: the number of [marked] cards [{in | of} <stackRef>]
 
 Syntax: the number of {char[acters] | items | words | lines} {in | of} <textString>
 
+Syntax: the number of elements {in|of} <array>
```

Comment:
Can you put spaces in here either side of the separator (`{in | of}`) please?

Reply:
Also we should probably avoid parameters that are the same as type names, perhaps use `<arrayExpr>` instead?